tests/modules.config

```diff
@@ -0,0 +1,238 @@
+    withName: PICARD_COLLECTRNASEQMETRICS {
+        ext.args       = '--STRAND_SPECIFICITY NONE'
+        publishDir     = [
+            [
+                path: "${params.outdir}/QC/Picard",
+                mode: params.publish_dir_mode,
+            ]
+        ]
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    withName: PRESEQ_LCEXTRAP {
+        ext.args       = {
+            [
+                '-verbose',
+                '-bam',
+                '-defects',
+                '-seed 1',
+                '-seg_len 10000000',
+            ].join(' ').trim()
+        }
+        publishDir     = [
+            [
+                path: "${params.outdir}/QC/lcextrap",
+                mode: params.publish_dir_mode,
+            ]
+        ]
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    withName: 'RSEQC_(BAMSTAT|INNERDISTANCE|INFEREXPERIMENT|JUNCTIONANNOTATION|JUNCTIONSATURATION|READDISTRIBUTION|READDUPLICATION)' {
+
+        publishDir     = [
+            [
+                path: "${params.outdir}/QC/rseqc",
+                mode: params.publish_dir_mode,
+            ]
+        ]
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    // TODO: retrieve dynamic resources samtools
+    withName: SAMTOOLS_CONVERT {
+
+        publishDir     = [
+            [
+                path: "${params.outdir}/cram_files/",
+                mode: params.publish_dir_mode,
+                pattern: '*.cra?',
+            ]
+        ]
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    withName: SAMTOOLS_FLAGSTAT {
+
+        publishDir     = [
+            [
+                path: "${params.outdir}/QC/samtools",
+                mode: params.publish_dir_mode,
+            ]
+        ]
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    withName: SAMTOOLS_IDXSTATS {
+
+
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    withName: SAMTOOLS_INDEX {
+
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    withName: SAMTOOLS_MERGE {
+
+        ext.args       = '--write-index'
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+    withName: SAMTOOLS_STATS {
+
+        cpus           = { 1  }
+        time           = { 10.m}
+        memory         = { 5.GB}
+    }
+
+    // TODO: retrieve dynamic resources sortmerna
+    withName: SORTMERNA_INDEX {
+
+        container      = 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/15/15b50826336c394eab2bf6d731f00729921598ffc6463278da6316e768b16b8b/data'
```

Comment:
Just out of interest, why do we override the SORTMERNA container? It is the same as in the 'production' config so it is correct for testing.

Reply:
Lets leave it for now ;)